.github/workflows/buildMod.yml

```diff
@@ -0,0 +1,18 @@
+name: '[CI] Build Mod'
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest 

```

Comment:
Windows pls

Reply:
Windows won't work, as you can see here: https://github.com/gruppe-adler/grad_meh/pull/2/commits/6b6514dc265f941c29b2d42f7e59c1086e4c785e